common/src/Assets/Texture.h

```diff
@@ -70,7 +72,10 @@ namespace TrenchBroom {
             bool isPrepared() const;
             void prepare(GLuint textureId, int minFilter, int magFilter);
             void setMode(int minFilter, int magFilter);
-            
+
+            void setFormat(GLenum newFormat);
```

Comment:
These methods are not necessary I think. The format should be set using the constructor, and it should not be necessary to read it from the outside.